### PR TITLE
Add geometry page block documentation

### LINKED ISSUE #380: Document the new geometry page block
The documentation should go here (reference) and here (configuration)

- [ ] Add to the page block reference (https://github.com/cortezaproject/corteza-docs/blob/2023.3.x/src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc, https://docs.cortezaproject.org/corteza-docs/2022.9/integrator-guide/compose-configuration/page-blocks.html). Use past page blocks as an example on how it should look like, what contents to write.

---

- [ ] Provide example (if no solid example pops to mind, you can ignore this one) -- add a few examples that come to mind, no need to go overboard (https://github.com/cortezaproject/corteza-docs/tree/2023.3.x/src/modules/integrator-guide/pages/compose-configuration/examples, https://docs.cortezaproject.org/corteza-docs/2022.9/integrator-guide/compose-configuration/examples/index.html)
* Make a new file
* include it in the index.adoc https://github.com/cortezaproject/corteza-docs/blob/2023.3.x/src/modules/integrator-guide/pages/compose-configuration/examples/index.adoc
* include it in the nav.adoc https://github.com/cortezaproject/corteza-docs/blob/2023.3.x/src/modules/integrator-guide/nav.adoc

Perhaps take a quick [look here](https://docs.cortezaproject.org/corteza-docs/2022.9/integrator-guide/automation/workflows/examples/index.html) for some inspiration.

### PR DESCRIPTION
**Pull request by kelanik8**
Geometry page block documentation

### CODE REVIEW

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 1]
```diff
@@ -743,3 +743,31 @@ Defines the report the page bock will use when showing data.
...
+image:compose-configuration/page-block-reference/config-geometry.png[...,49%,role="preview data-zoomable"]
+image:compose-configuration/page-block-reference/added-geometry.png[...,49%,role="preview data-zoomable"]
+
+.Additional configurations:
+[cols="2s,5a"]
+|===
+| [#page-block-zoom-min]#<<page-block-zoom-min,Zoom min>>#
+|
+Defines the minimum zoom level of the map.
+
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc:761]: just one empty line please

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 2]
```diff
@@ -743,3 +743,31 @@ Defines the report the page bock will use when showing data.
...
+| [#page-block-zoom-max]#<<page-block-zoom-max,Zoom max>>#
+|
+Defines the maximum zoom level of the map.
+
+| [#page-block-zoom-lockbounds]#<<page-block-zoom-lockbounds,Lock bounds>>#
+|
+Defines if the map should be locked to the bounds of the marker.
+
+|===
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc:770]: empty line at the end please

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 3]
```diff
@@ -743,3 +743,31 @@ Defines the report the page bock will use when showing data.
 Defines the display element from the selected report the page block will use when showing data.
 
 |===
+
+[#page-block-geometry]
+== Geometry
+
+The *geometry page block* allows you to show a map with a marker, lines and polygons from different geometry values of different record sources which are rendered on the map.
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc]: `...map with a marker, lines...` -> `...map with markers, lines...`

Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc]: `...record sources which are rendered on the map.` -> `...record sources.`

The being rendered on the map is already explained in the first bit -- `...allows you to show a map with...`

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 4]
```diff
@@ -743,3 +743,31 @@ Defines the report the page bock will use when showing data.
...
+
+Geometry page block can render a map within certain map constraints.
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc]: I don't understand this part.

**Review by tjerman — CHANGES_REQUESTED** [review]

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 5]
```diff
@@ -743,3 +743,28 @@ Defines the report the page bock will use when showing data.
...
+|
+Defines the minimum zoom level of the map.
+
+| [#page-block-zoom-max]#<<page-block-zoom-max,Zoom max>>#
+|
+Defines the maximum zoom level of the map.
+
+| [#page-block-zoom-lockbounds]#<<page-block-zoom-lockbounds,Lock bounds>>#
+|
+Defines if the map should be locked to the bounds of the marker.
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc]: Should this be explained a bit more? I'm not sure what exactly this is about.
In the bounds of the displayed pins?

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 6]
```diff
@@ -743,3 +743,28 @@ Defines the report the page bock will use when showing data.
...
 
 |===
+
+[#page-block-geometry]
+== Geometry
+
+The *geometry page block* allows you to show a map with markers, lines and polygons from different geometry values of different record sources.
+
+image:compose-configuration/page-block-reference/config-geometry.png[...,49%,role="preview data-zoomable"]
+image:compose-configuration/page-block-reference/added-geometry.png[...,49%,role="preview data-zoomable"]
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc:753]: Replace with screenshots that follow the guidelines

**Review by tjerman — CHANGES_REQUESTED** [review]

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc [hunk 7]
```diff
@@ -743,3 +743,28 @@ Defines the report the page bock will use when showing data.
...
+|
+Defines the minimum zoom level of the map.
+
+| [#page-block-zoom-max]#<<page-block-zoom-max,Zoom max>>#
+|
+Defines the maximum zoom level of the map.
+
+| [#page-block-zoom-lockbounds]#<<page-block-zoom-lockbounds,Lock bounds>>#
+|
+Defines if the map should be prevented from being panned out or zoomed out of the current rectangular area of the map.
```
Comment (tjerman) [inline src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc]: What are the bounds defined by?

**Review by tjerman — CHANGES_REQUESTED** [review]
Your screenshots still don't follow [the guidelines](https://docs.cortezaproject.org/corteza-docs/2023.3/developer-guide/documentation/annotations.html)

File: src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc (hunk 7 above)
Comment (kelanik8) [reply src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc]: lat and lng

**Review by tjerman — APPROVED** [review]
lgtm; good job

**Pull request merged**